msg/vehicle_odometry.msg

```diff
@@ -56,4 +56,4 @@ float32 yawspeed		# Angular velocity about Z body axis
 # If angular velocity covariance invalid/unknown, 16th cell is NaN
 float32[21] velocity_covariance
 
-# TOPICS vehicle_odometry vehicle_mocap_odometry vehicle_visual_odometry vehicle_aligned_visual_odometry
+# TOPICS vehicle_odometry vehicle_mocap_odometry vehicle_visual_odometry vehicle_visual_odometry_aligned
```

Comment:
What if mocap is not aligned?

Reply:
Are people still using it? At the moment it is used only in LPE and Attitude estimator.

Reply:
We can just call it `vehicle_odometry_aligned`. I don't think anyone will be using mocap and VO simultaneously as a source of odometry. Maybe as groundtruth, but that is beyond the scope of this PR.

Reply:
`vehicle_odometry_aligned` will be messed with the `vehicle_odometry`, which is curre tly being used as the feedback of the estimator. Let's keep it this way for now.